portal-api/api-tests/v1/gef/facilities/facilities.delete.api-test.js

```diff
@@ -91,7 +91,7 @@ describe(baseUrl, () => {
       collectionName: MONGO_DB_COLLECTIONS.FACILITIES,
       auditRecord: expectAnyPortalUserAuditDatabaseRecord(),
       getDeletedDocumentIds: () => facilitiesToDeleteIds,
-      expectedSuccessResponseBody: { acknowledged: true },
+      expectedSuccessResponseBody: { acknowledged: true, deletedCount: 2 },
```

Comment:
?

Reply:
My change to `deleteMany` meant it didnt return the deletedCount, on the grounds of YAGNI. Without transactions this was much easier to do so I've had to add it to the response body here. This was the original behaviour pre-audit logs